committee/db/base/signrequest.go

```diff
@@ -18,8 +18,9 @@ type VerificationRequest struct {
 	TXHash string `gorm:"column:tx_hash;index;size:256"`
 	// TransactionID is the id of the transaction
 	TransactionID string `gorm:"column:transaction_id;index;size:256;primaryKey"`
-	// DataHash is the hash of the data
-	Entry string `gorm:"column:entry"`
+	// Entry is the entry to be signed
+	// TODO: consider renaming to batch
```

Comment:
Let's rename this? Feels like the DB will be dropped anyway, so it shouldn't be a blocker.